game/games/pixel_side_scroller/objects/spikes.tscn

```diff
@@ -0,0 +1,102 @@
+[gd_scene load_steps=7 format=2]
+
+[ext_resource path="res://shared/fonts/press_start2p/press_start2p_regular.ttf" type="DynamicFontData" id=1]
+[ext_resource path="res://games/pixel_side_scroller/entitys/damage_trigger.tscn" type="PackedScene" id=2]
+[ext_resource path="res://games/pixel_side_scroller/sounds/sfx/trap_activate.wav" type="AudioStream" id=3]
+
+[sub_resource type="DynamicFont" id=4]
+size = 8
+font_data = ExtResource( 1 )
+
+[sub_resource type="GDScript" id=5]
+script/source = "extends Label
+
+export(bool) var toggle_spikes = false
+export(bool) var initial_state = false
+export(float) var toggle_time_on = 1
+export(float) var toggle_time_off = 1
+export(float) var start_offset_time = 0
+
+
+# var _triggered: bool = false
+var _toggle_state = false
```

Comment:
I don't like this name. If I look at this I don't know what true and what false means.
Maybe convert this to an enum that can be Spike or ground or just name it _is_currently_spike

Reply:
I kind of agree, "toggle state" is kind of like calling it "boolean". How about simply… `_active`?

Reply:
`_active` is probably better than an enum

Reply:
Good Points :) will fix tomorrow when I have time